docs/commands.md

```diff
@@ -0,0 +1,86 @@
+<!--[metadata]>
++++
+title = "Understand the typical commands"
+description = "Description of key notary commands"
+keywords = ["docker, notary, notary-client"]
++++
+<![end-metadata]-->
+
+# Understand the typical notary commands
+
+## Initialize a collection
+`notary init example.com/image`
+
+`notary publish example.com/image`
+
+## Add delegations
+`notary delegation add example.com/image targets/qa_team --all-paths <key>`
+
+Note: This will be staged as a changelist, you will need to publish for it to take effect
```

Comment:
Not sure it makes sense to call this out after every command. Maybe there should be a general doc somewhere that describes that at a high level how notary makes local changes that eventually need to be published?